packages/@sanity/form-builder/src/inputs/Array/ItemValue.js

```diff
@@ -162,6 +163,24 @@ export default class RenderItemValue extends React.Component<Props> {
       )
     }
 
+    const itemType = type.of.find(i => i.name === item._type)
+
+    if (itemType.fields.length > 1) {
+      return (
+        <div className={styles.popupAnchor}>
+          <DefaultDialog
+            padding={true}
+            isOpen
+            title={item.title}
+            onClose={this.handleEditStop}
+            key={item._key}
+          >
+            {content}
+          </DefaultDialog>
+        </div>
+      )
+    }
+
```

Comment:
@bjoerge should probably review this, I'm not up to speed on whether or not this will work in all cases

Reply:
LGTM